issue.go

```diff
@@ -499,7 +499,7 @@ func (j *Jira) IssueRemoveWatcher(issue, user string) error {
 
 func IssueRemoveWatcher(ua HttpClient, endpoint string, issue, user string) error {
 	uri := URLJoin(endpoint, "rest/api/2/issue", issue, "watchers")
-	uri += fmt.Sprintf("?accountId=%s", user)
+	uri += fmt.Sprintf("?username=%s", user)
```

Comment:
Per the docs:
```
username
string
This parameter is no longer available. See the deprecation notice for details.
```
Jira is transitioning wholesale to the (sadly opaque) `accountId`.

Reply:
Or something. When I was first starting out with go-jira this was a new issue, but the way the Jira ecosystem is, we probably need a better solution. Could we a) set up a maintainer team or b) enable discussions here - I think this is a bigger design issue that we can usefully resolve in PR comments.